src/github.com/getlantern/launcher/launcher_windows.go

```diff
@@ -0,0 +1,38 @@
+// Package launcher configures Lantern to run on system start
+package launcher
+
+import (
+	"github.com/kardianos/osext"
+	"github.com/luisiturrios/gowin"
+
+	"github.com/getlantern/golog"
+)
+
+const (
+	RunDir = `Software\Microsoft\Windows\CurrentVersion\Run`
+)
+
+var (
+	log = golog.LoggerFor("launcher")
+)
+
+func CreateLaunchFile(autoLaunch bool) {
+	var err error
+
+	if autoLaunch {
+		lanternPath, err := osext.Executable()
+		if err != nil {
+			log.Errorf("Could not get Lantern directory path: %q", err)
+			return
+		}
+		err = gowin.WriteStringReg("HKCU", RunDir, "value", lanternPath)
```

Comment:
These registry calls still succeed if not running as admin, is that right? Might be worth testing with a full installer.